lib/galaxy/jobs/__init__.py

```diff
@@ -1527,7 +1527,17 @@ def path_rewriter(path):
         self.cleanup(delete_files=delete_files)
 
     def check_tool_output(self, stdout, stderr, tool_exit_code, job):
-        return check_output(self.tool, stdout, stderr, tool_exit_code, job)
+        job_id_tag = "<unknown job id>"
+        if job is not None:
+            job_id_tag = job.get_id_tag()
+
+        state, trimmed_stdout, trimmer_stderr = check_output(self.tool.stdio_regexes, self.tool.stdio_exit_codes, stdout, stderr, tool_exit_code, job_id_tag)
```

Comment:
What a random typo... also I don't think this actually trims those outputs. I need work on this.